### PR TITLE
docs(client-examples): add transfer to readme

### PR DESCRIPTION
**Pull request by javadkh2**

### CODE REVIEW

File: packages/libs/client-examples/README.md [hunk 1]
```diff
@@ -1,17 +1,155 @@
...
-This project demonstrates the use of the `@kadena/pact-cli` together with
-`@kadena/client` for _smart contracts_.
+```shell
+npx pactjs contract-generate --contract="coin" --api http://104.248.41.186:8080/chainweb/0.0/development/chain/0/pact
+```
+
+### Creating a type definition from a pact file
+
+```shell
+npx pactjs contract-generate --file=./coin.pact --api http://104.248.41.186:8080/chainweb/0.0/development/chain/0/pact
```
Comment (EnoF) [inline packages/libs/client-examples/README.md:26]: When I tried this, (with testnet) it gave me an error, the combination of `--file` and `--api` not sure if it's already fixed though (apha.7)

File: packages/libs/client-examples/README.md [hunk 2]
```diff
@@ -1,17 +1,155 @@
...
+  // - sigs: an array that has the same length as signers in the command but all filled by undefined
+
+  // Now you need to sign the transaction; you can do it in a way that suits you.
+  // We exported some helpers like `signWithChainweaver` and signWithWalletConnect, but you can also use other wallets.
+  // For example, if you are using EckoWallet, it has a specific API for signing transactions.
+  // By the end, the signer function should fill the sigs array and return the signed transaction.
+  const signedTr = await signWithChainweaver(transaction);
+
+  // As the signer function could be an external function, we double-check if the transaction is signed correctly.
+  if (isSignedCommand(signedTr)) {
```
Comment (EnoF) [inline packages/libs/client-examples/README.md:115]: Even though it's example code, I'd still recommend inverting the if statement to get early returns instead.

Comment (EnoF) [reply packages/libs/client-examples/README.md:115]:
```
if (!isSignedCommand(signedTr)) throw new Error('Transaction has not been signed');

... other code ...
if (response.result.status === 'failure) throw response.result.error;

console.log('yaaaaay')
```

**Review by EnoF — APPROVED** [review]

File: packages/libs/client-examples/README.md (hunk 1 above)
Comment (ash-vd) [reply packages/libs/client-examples/README.md:26]: This error is expected, you don't need an API when you generate from a file. On a related note I'm wondering where the IP address comes from?

**Comment by javadkh2** [comment]
I am updating the doc

File: packages/libs/client-examples/README.md (hunk 1 above)
Comment (javadkh2) [reply packages/libs/client-examples/README.md]: that was the devnet url, I have replaced it with the mainnet url.
I also explained when you need --api and when you don't

File: packages/libs/client-examples/README.md (hunk 2 above)
Comment (javadkh2) [reply packages/libs/client-examples/README.md]: Done!

**Comment by javadkh2** [comment]
@EnoF @alber70g I have updated the readme, can you please have a look
and could you please also read the code comments.

**Review by alber70g — APPROVED** [review]

**Pull request merged**